<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sesion</class>
 <widget class="QDialog" name="Sesion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblMensaje">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingresa tu usuario:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUsuario">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingresa Pin:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIngresar">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingresar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>261</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="Consultar">
        <property name="text">
         <string>Consultar Saldo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Depositar">
        <property name="text">
         <string>Depositar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Retirar">
        <property name="text">
         <string>Retirar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CambiarPin">
        <property name="text">
         <string>Cambiar Pin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Salir">
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>Consultar</zorder>
     <zorder>Retirar</zorder>
     <zorder>CambiarPin</zorder>
     <zorder>Salir</zorder>
     <zorder>Depositar</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
